### PR TITLE
Add JSON literal alias tests.

### PR DESCRIPTION
**Pull request by davidlehn**
Tests for an implementation issue found in jsonld.js:
https://github.com/digitalbazaar/jsonld.js/pull/346

### CODE REVIEW

File: tests/expand/js14-in.jsonld [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "@context": {
+    "@version": 1.1,
```
Comment (dlongley) [inline tests/expand/js14-in.jsonld]: Do we need an explicit version here?

Comment (davidlehn) [reply tests/expand/js14-in.jsonld]: I was matching the style of the other js* tests.

**Comment by gkellogg** [comment]
Technically, it’s a 1.1 test, so `@version` is appropriate, but no longer necessary.

**Comment by gkellogg** [comment]
Also, expansion tests should be replicated in toRdf, as @rubensworks does not go through an intermediate expansion step; this has been the pattern for other tests.

**Comment by gkellogg** [comment]
For #237

**Pull request merged**